tools/expotools/tsconfig.json

```diff
@@ -15,6 +15,9 @@
     "noFallthroughCasesInSwitch": true,
     "esModuleInterop": true,
     "resolveJsonModule": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "skipDefaultLibCheck": true,
+    "incremental": true,
+    "tsBuildInfoFile": "./cache/.tsbuildinfo"
```

Comment:
I noticed some weird behavior with `.tsbuildinfo` and `incremental` builds. If you remove a built file, without touching the `.tsbuildinfo`, it wouldn't rebuild that file. Could that be a problem here as well?

Reply:
Yeah, I've noticed this too, but tbh you shouldn't ever need to remove build folder 😄 Not sure whether we can do anything with this without time overhead 🤷‍♂️